src/io/flutter/run/FlutterRunningState.java

```diff
@@ -106,4 +117,25 @@ public static VirtualFile flutterProjectDir(@Nullable String projectPath) throws
   public int getObservatoryPort() {
     return myObservatoryPort;
   }
+
+  private class OpenIOSSimulatorLink implements HyperlinkInfo {
+    @Override
+    public void navigate(Project project) {
+      final GeneralCommandLine cmd = new GeneralCommandLine().withExePath("open").withParameters("-a", "Simulator.app");
+      try {
+        final OSProcessHandler handler = new OSProcessHandler(cmd);
+        handler.addProcessListener(new ProcessAdapter() {
+          @Override
+          public void processTerminated(final ProcessEvent event) {
+            //TODO: schedule a re-run or notify user to try again.
```

Comment:
Good idea.  I added a new issue (https://github.com/flutter/flutter-intellij/issues/127) and updated the TODO.